tests/unit/amo/components/TestAddonsByAuthorsCard.js

```diff
@@ -66,24 +68,19 @@ describe(__filename, () => {
 
   function addonsWithAuthorsOfType({ addonType, multipleAuthors = false }) {
     return loadAddonsByAuthors({
-      addons: [
-        {
+      addons: Array(
+        addonType === ADDON_TYPE_THEME ?
+          THEMES_BY_AUTHORS_PAGE_SIZE :
+          EXTENSIONS_BY_AUTHORS_PAGE_SIZE
+      )
+        .fill('')
```

Comment:
Heh, what does this code do exactly? I think a `for` would be more readable.

Reply:
there is `fill` with an empty string in between 😉

Reply:
Yes that's deliberate, `Array(<number>)`, creates array with X empty slots, `fill('')` creates dummy value for each slots, and `map` actually inserts the value in it.
I just don't want to use `for` loop.

Reply:
I get that, but it is tricky to grasp because it is unusual.

Reply:
But isn't that okay?

Reply:
I would like to see a simpler way of doing this, `Array(N).fill('').map()` is not explicit at all, think about newcomers 😉

Reply:
We don't want to see such hacks in our codebase, I am sorry. I like ES6/Next syntax a lot, but this is not obvious at all. A simple `for` loop before the `return` would be perfect.